tests/schemas/test_identifier_schema.py

```diff
@@ -233,3 +233,48 @@ def test_invalid_custom_inheritance_not_required():
 
     with pytest.raises(ValidationError) as excinfo:
         loaded = CustomRequiredSchema().load(invalid_custom)
+
+
+class CustomUnkownAcceptedSchema(IdentifierSchema):
```

Comment:
moderate: Are these two classes strictly necessary? I think it can be tested directly on the `IdentifierSchema`? The indication would be the fact that there is no need for the `extra` attribute.